ibazel/profiler/profiler.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function(){
+  function extractUrl() {
+    var url;
+    [...window.document.getElementsByTagName('script')].forEach(function (element) {
```

Comment:
Can you run this through clang format? I expect this line to be:

```
[...window.element..........]
    .forEach(function(element) {
  if (!url && element.src
```

and a similar rendering on line 19. I defer to clang-format so if it disagrees with me then ship that.

Reply:
👍